redskyctl/internal/commands/export/export_api_test.go

```diff
@@ -106,7 +106,11 @@ func (f *fakeRedSkyServer) GetExperiment(ctx context.Context, name string) (expe
 	return experimentsapi.Experiment{}, nil
 }
 
-func (f *fakeRedSkyServer) CreateExperiment(ctx context.Context, name experimentsapi.ExperimentName, exp experimentsapi.Experiment) (experimentsapi.Experiment, error) {
+func (f *fakeRedSkyServer) CreateExperimentByName(ctx context.Context, name experimentsapi.ExperimentName, experiment experimentsapi.Experiment) (experimentsapi.Experiment, error) {
```

Comment:
Does the `fakeRedSkyServer` here need to be updated to `fakeOptimizeServer` or similar?

Reply:
I'll get that with the 0.0.10 update when I'm doing the bulk of the type renaming.